.github/workflows/multichain-e2e-template.yml

```diff
@@ -114,6 +114,10 @@ jobs:
         run: make register-bank-assets
         working-directory: ./agoric-sdk/multichain-testing
 
+      - name: Create Osmosis Pool
+        run: make create-osmosis-pool
+        working-directory: ./agoric-sdk/multichain-testing
+
```

Comment:
Nice work. Instead invoking this from the GH workflow, can we instead use `ava` to call the script? This way we only run this set up for a test that will actually use it.